core/templates/dev/head/collection_editor/collection_editor.html

```diff
@@ -80,5 +173,10 @@
     {{ include_js_file('domain/editor/ChangeObjectFactory.js') }}
     {{ include_js_file('domain/editor/UndoRedoService.js') }}
     {{ include_js_file('domain/utilities/UrlInterpolationService.js') }}
+    {{ include_js_file('editor/ExplorationEditor.js') }}
```

Comment:
Hi @BenHenning, sorry for responding with such delay. I think that creating a new RouterService is a really good idea! We should limit it's responsibilities to only track currently opened tab (or something similar). Also I think that in a new RouterService we should take advantage of awesome [Angular-UI Router](https://github.com/angular-ui/ui-router) to not scan URL for figuring out which tab is opened. What do you think?